language-support/ts/daml-ledger/index.test.ts

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Template, Choice, ContractId } from "@daml/types";
+import Ledger, {CreateEvent} from "./index";
+import { Event } from "./index";
+import * as jtv from "@mojotech/json-type-validation";
+import { EventEmitter } from 'events';
```

Comment:
We only use the types from this package and import the values further down using `require`. Let's make that explicit.
```suggestion
import type { EventEmitter } from 'events';
```